From 1b5cddb7e19675a4d4b965826f3926759b197b4e Mon Sep 17 00:00:00 2001
From: Stuart Menefy <stuart.menefy@codasip.com>
Date: Tue, 16 Jan 2024 10:49:54 +0000
Subject: [PATCH 07/12] configs: codasip-a70x-hobgoblin: Move environment file
 to first partition

To support having the SD card partitioned (so that we can have a Linux
file system on the second partition) move the location of the
environment file to the first partition.

Signed-off-by: Stuart Menefy <stuart.menefy@codasip.com>
---
 configs/codasip-a70x-hobgoblin_smode_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/codasip-a70x-hobgoblin_smode_defconfig b/configs/codasip-a70x-hobgoblin_smode_defconfig
index a0c4cf8098..59fb6ac48d 100644
--- a/configs/codasip-a70x-hobgoblin_smode_defconfig
+++ b/configs/codasip-a70x-hobgoblin_smode_defconfig
@@ -2,6 +2,7 @@ CONFIG_RISCV=y
 CONFIG_NR_DRAM_BANKS=1
 CONFIG_HAS_CUSTOM_SYS_INIT_SP_ADDR=y
 CONFIG_CUSTOM_SYS_INIT_SP_ADDR=0x80200000
+CONFIG_DM_GPIO=y
 CONFIG_DEFAULT_DEVICE_TREE="codasip-a70x"
 CONFIG_SYS_LOAD_ADDR=0x80200000
 CONFIG_TARGET_CODASIP_A70X=y
@@ -31,7 +32,7 @@ CONFIG_CMD_TIMER=y
 CONFIG_OF_BOARD=y
 CONFIG_OF_OMIT_DTB=y
 CONFIG_ENV_IS_IN_FAT=y
-CONFIG_ENV_FAT_DEVICE_AND_PART="0"
+CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
 CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_OCORES=y
 # CONFIG_INPUT is not set
-- 
2.39.3

